universite-du-quebec-a-montreal.csl

```diff
@@ -147,29 +184,60 @@
                   <text variable="URL" prefix="de "/>
```

Comment:
also not yours originally, but what is the "note" doing here? We like to avoid that.

Reply:
I will avoid "note" as much as possible.